platform/ios/PolyPodApp/PodApi/PolyOut/FS/PolyOut+FS.swift

```diff
@@ -43,17 +43,17 @@ extension PolyOut {
         return documentDirectory.appendingPathComponent(PolyOut.fsFilesRoot).appendingPathComponent(activeFeature!.id)
     }
     
-    private func pathFromUrl(url: String) -> URL? {
-        let id = idFromUrl(url: url)
-
-        return id != nil ? featureFilesPath().appendingPathComponent(id!) : nil
+    func fsUriFromId(_ id: String) -> URL {
+        return featureFilesPath().appendingPathComponent(id)
     }
     
-    func pathFromId(id: String) -> URL {
-        return featureFilesPath().appendingPathComponent(id)
+    private func fsUriFromPodUrl(_ url: String) -> URL? {
+        let id = idFromPodUrl(url)
+
+        return id != nil ? featureFilesPath().appendingPathComponent(id!) : nil
     }
     
-    private func idFromUrl(url: String) -> String? {
+    private func idFromPodUrl(_ url: String) -> String? {
```

Comment:
Since you're renaming anyway, is this really appropriate to be called `id`? Isn't it more like.... `relativePath`? But I can't say I _fully_ understand this code and all these conversions :D

Reply:
The result of this method is a `UUID`. I think this is okay.